sys-fs/btrfsmaintenance/btrfsmaintenance-0.4.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd
+
+DESCRIPTION="Scripts for btrfs maintenance tasks like periodic scrub, balance, trim or defrag"
+HOMEPAGE="https://github.com/kdave/btrfsmaintenance"
+SRC_URI="https://github.com/kdave/btrfsmaintenance/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="systemd"
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	sys-fs/btrfs-progs
+	virtual/cron"
+
+src_prepare() {
+	# Fix config path into watching service
+	sed -i 's%/etc/sysconfig/btrfsmaintenance%/etc/default/btrfsmaintenance%g' btrfsmaintenance-refresh.* || die "Unable to patch btrfsmaintenance-refresh.*"
+	default_src_prepare
+}
+
+src_install() {
+	dodoc README.md CONTRIBUTING.md btrfsmaintenance.changes
+	insinto /etc/default
+	newins sysconfig.btrfsmaintenance btrfsmaintenance
+	insinto /usr/share/btrfsmaintenance
+	doins btrfsmaintenance-functions
+	exeinto /usr/share/btrfsmaintenance
+	doexe btrfs*.sh
+	# Fix config path into watching service
+	sed -i 's%/etc/sysconfig/btrfsmaintenance%/etc/default/btrfsmaintenance%g' btrfsmaintenance-refresh.path
```

Comment:
Break up this line.

Reply:
We needn't because this job is already done by src_prepare()